vendor/composer/autoload_static.php

```diff
@@ -61,6 +61,10 @@ class ComposerStaticInit22ddb69348c7ed922c96325249cef3d0
         array (
             'Psr\\Log\\' => 8,
```

Comment:
Getting closer :)
 Appears you did not remove the phing stuff. See here for instructions on removing the phing stuff(step 4 and 5):
http://www.open-emr.org/wiki/index.php/Composer#Migrating